### ISSUE TITLE
Code not compiling

### ISSUE THREAD

**Issue by Domosed-gamin** [ro-i/cargame#1]
Hi man! I written make in terminal and game is not build. If you have spare time, please check out and fix this error.

```
ringo@qwerty:~/Загрузки/cargame-development$ make
gcc -std=c99 -pedantic -Wall -Wextra -O2 -I. -I/usr/include -D_XOPEN_SOURCE -D_XOPEN_SOURCE_EXTENDED -D_DEFAULT_SOURCE  -I/usr/include/ncursesw  -DVERSION_STR=\"0.1.0\" -DNAME_STR=\"cargame\" -c -o cargame.o cargame.c
cargame.c:167:36: error: initializer element is not constant
 static Game    game = { .borders = default_borders };
                                    ^~~~~~~~~~~~~~~
cargame.c:167:36: note: (near initialization for ‘game.borders’)
Makefile:39: ошибка выполнения рецепта для цели «cargame.o»
make: *** [cargame.o] Ошибка 1

```

**Comment by ro-i** [comment]
That's interesting, because I've never get this error with the current code - neither with gcc `gcc (Debian 8.3.0-6) 8.3.0`, nor with clang `clang version 7.0.1-8 (tags/RELEASE_701/final)`.
But I think that commit `f31f74c9d8ed02125c27f1370fb97815cb727187` (https://github.com/ro-i/cargame/commit/f31f74c9d8ed02125c27f1370fb97815cb727187) fixes the problem for you.
Please confirm if the issue could be fixed! Which version of gcc do you use?

**Comment by Domosed-gamin** [comment]
```
ringo@qwerty:~/Загрузки/cargame-development$ gcc --version
gcc (Debian 6.3.0-18+deb9u1) 6.3.0 20170516
Copyright (C) 2016 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```
I tried to compile new version and I have same error.

**Comment by ro-i** [comment]
I could reproduce the issue with your gcc version. It is now fixed by commit `d77a5d3bb0e74b31e62e33e5c03c1addf5edb437` (https://github.com/ro-i/cargame/commit/d77a5d3bb0e74b31e62e33e5c03c1addf5edb437).
The issue is related to: https://gcc.gnu.org/onlinedocs/gcc/Compound-Literals.html
> As a GNU extension, GCC allows initialization of objects with static storage duration by compound literals (which is not possible in ISO C99 because the initializer is not a constant).

I don't know, why newer gcc versions do neither throw this error, nor throw a warning, even with the flags I use for all my code (`-std=c99 -pedantic -Wall -Wextra`).
Anyway - thank you for your report!
